### DISCUSSION TITLE
Question: What's an adjective to describe Rant Lang / what category of software does this fall under?

### DISCUSSION THREAD

**Discussion by YousufSSyed** [rant-lang/rant#12] [Q&A]
It says on the readme, "Rant is a dynamically-typed, multi-paradigm templating language designed primarily for procedural generation." So, would calling it a text templating language be right, or is there another phrase that describes it better and or in a more common way? I never heard of software like this, what other pieces of software are there are comparable to Rant lang?

**Comment by TheBerkin** [comment]
I would call it a data templating language. Rant can also work with data structures other than strings.

**Reply by YousufSSyed** [reply]
Can you think of phrases I can google to find software similar to Rant? I'm just curious if there even are any.

**Reply by TheBerkin** [reply]
Honestly not sure if there's anything else with exactly the same feature set as Rant, but you could try "data templating language" or "procedural generation language" and go from there.

**Reply by YousufSSyed** [reply]
I tried those but thanks anyway.
